### DISCUSSION TITLE
How to use 3rd party SDK in ionic Capacitor project?

### DISCUSSION THREAD

**Discussion by Saqib92** [ionic-team/capacitor#5609] [Show and tell]
How to use 3rd party SDK into Ionic Capacitor project? i am looking to use [https://www.xsens.com/](https://www.xsens.com/) SDK into my ionic application based on Capacitor. should make a plugin for this? or is there an easy way available to use SDKs ?

**Comment by w-mazed** [comment]
Hi,
I've faced the same situation, so I created a custom plugin, I think there is no way to deal with a 3rd party lib, exception creating your own plugin :
https://capacitorjs.com/docs/plugins/creating-plugins

Best regards.

**Reply by Saqib92** [reply]
Yes, I already kneeled down to this. I am working on my own plugin for this. Thanks.

**Reply by bohdan-tomchuk** [reply]
@Saqib92 can you share your implementation ?

**Reply by Saqib92** [reply]
My plugin is for my client hence it's not public so can't share it. you can check capacitor plugin docs on how to create plugin. thanks

**Reply by bohdan-tomchuk** [reply]
@Saqib92 i have troubles with implementing 3rd party sdk in custom capacitor plugin. Docs show only work with native apis(((
